samples/apps/spring-petclinic/build-app.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
should not we wrap this logic inside of a `Dockerfile`, so that we do not need to prepare build environment locally. I think you had this previously?

Reply:
indeed, I'll do it eventually, but this builds much faster so it allows me going forward faster while I'm still working on it.

Reply:
done